lib/authOptions.js

```
@@ -18,15 +18,15 @@ export const authOptions = {
           await dbConnect();
 
           if (
-            credentials.email === bypassCredentials.email &&
-            credentials.password === bypassCredentials.password
+            credentials.email === credentials.email &&
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)